packages/fx-core/tests/plugins/resource/aad/unit/index.test.ts

```diff
@@ -128,63 +119,6 @@ describe("AadAppForTeamsPlugin: CI", () => {
     const postProvision = await plugin.postLocalDebug(context);
     chai.assert.isTrue(postProvision.isOk());
   });
-
-  it("provision and local debug: tab and bot with update current and wrong permission", async function () {
```

Comment:
why test case is deleted?

Reply:
These test case are for test UpdatePermission Command which are not deleted previously.